its/projects/TwoDrivesTwoProjects/TwoDrivesTwoProjects.sln

```diff
@@ -5,7 +5,7 @@ VisualStudioVersion = 17.2.32526.322
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DefaultDrive", "DefaultDrive\DefaultDrive.csproj", "{5B64B1C0-C68F-4585-B02A-8789CD7C31CE}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DriveZ", "Z:\DriveZ.csproj", "{1B0529BA-29F3-49F1-9977-99050FAC907A}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DriveZ", "Z:\Subfolder\DriveZ.csproj", "{1B0529BA-29F3-49F1-9977-99050FAC907A}"
```

Comment:
Why this change? I dont' see `DriveZ.csproj` being in `SubFolder`

Reply:
I moved the project into a subfolder.

To give some context, when in the root drives the following error happens:
```
CSC : error CS5001: Program does not contain a static 'Main' method suitable for an entry point [Z:\DriveZ.csproj]
```

I moved it to `Subfolder` because we have a similar [test](https://github.com/SonarSource/sonar-scanner-msbuild/blob/master/its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java#L869-L889) with DriveY (see [here](https://github.com/SonarSource/sonar-scanner-msbuild/tree/master/its/projects/TwoDrivesThreeProjects)) that is in a subfolder and does not have this issue.